### PR TITLE
0.9.1

### PR DESCRIPTION
**Pull request by thomaslee**
Hopefully I did this right. Ancillary branches (`pristine-tar`) and tags (`upstream/0.9.1`) should be up to date.

Let's see if the PR build works against `debian/experimental`.

### CODE REVIEW

**Comment by thomaslee** [comment]
Should mention one surprise here is that I had to add an explicit dependency on `libssl-dev` in `libcapnp-dev`. I think this is correct in principle if you build capnp with SSL support enabled, or at least that's what the official cmake files imply. I'll take a closer look later.

Other than that it was a pretty easy import, refresh patches + update packaging for the new version.

**Comment by thomaslee** [comment]
(The `libssl-dev` thing was surfaced by the integration tests, specifically `debian/tests/build-tests`)

**Comment by thomaslee** [comment]
^^ Fixes a lintian error but think it introduces this warning https://lintian.debian.org/tags/distant-prerequisite-in-shlibs ... I'll look into that later.

**Comment by thomaslee** [comment]
Yep, the lintian warning was just me goofing up the shlibs file adding libcapnp-websocket.

File: debian/changelog [hunk 1]
```diff
@@ -1,7 +1,10 @@
-capnproto (0.8.0-3) UNRELEASED; urgency=medium
+capnproto (0.9.1-1) UNRELEASED; urgency=medium
 
   * Add serde-tests autopkgtest
   * Add files for CI automation (GitHub Actions)
+  * Imported Upstream version 0.9.1
+  * Update packaging for 0.9.2
```
Comment (tmancill) [inline debian/changelog]: 0.9.2 or 0.9.1?

**Review by tmancill — APPROVED** [review]
Just the one comment about the debian/changelog entry, and you should go ahead and bump the years in debian/copyright.  Otherwise, it's flawless.  :)

File: debian/changelog (hunk 1 above)
Comment (thomaslee) [reply debian/changelog]: 🤦 glad one of us is paying attention

**Pull request merged**